content/applications/finance/accounting/reporting/year_end.rst

```diff
@@ -2,115 +2,112 @@
+:guilabel:`Start Date` and :guilabel:`End Date`. The process should then be repeated for the
+following fiscal years.
 
 .. _year-end/checklist:
 
 Year-end checklist
 ==================
 
-- Run a **Tax report**, and verify that your tax information is correct.
-
-- Reconcile all accounts on your **Balance Sheet**:
+Before closure
+--------------
 
-  - Agree your bank balances in Odoo against your actual bank balances
-    on your statements. Utilize the **Bank Reconciliation** report to
-    assist with this.
+Before closing a fiscal year, first ensure everything is accurate and up-to-date:
 
-  - Reconcile all transactions in your cash and bank accounts by
-    running your **Aged Receivables** and **Aged Payables** reports.
+- Make sure all bank accounts are fully :doc:`reconciled <../bank/reconciliation>` up to year-end,
+  and confirm that the ending book balances match the bank statement balances.
+- Verify that all :doc:`customer invoices <../customer_invoices>` have been entered and
+  approved, and that there are no draft invoices.
+- Confirm that all :doc:`vendor bills  <../vendor_bills>` have been entered and agreed upon.
+- Validate all :doc:`expenses <../../expenses>`, ensuring their accuracy.
+- Corroborate that all :doc:`received payments <../payments>` have been encoded and recorded
+  accurately.
+- Close all :ref:`suspense accounts <bank_accounts/suspense>`.
+- Book all :doc:`depreciation <../vendor_bills/assets>` and :doc:`deferred revenues
+  <../customer_invoices/deferred_revenues>` entries.
 
-  - Audit your accounts, being sure to fully understand the
-    transactions affecting them and the nature of the
-    transactions, making sure to include loans and fixed assets.
+Closing a fiscal year
+---------------------
 
-- Run the optional **Payments Matching** feature, under the **More**
-  dropdown on the dashboard, validating any open **Vendor Bills** and
-  **Customer Invoices** with their payments. This step is optional,
-  however it may assist the year-end process if all outstanding
-  payments and invoices are reconciled, and could lead finding
-  errors or mistakes in the system.
+Then, to close the fiscal year:
 
-- Your accountant/bookkeeper will likely verify your balance sheet
-  items and book entries for:
+- Run a :ref:`tax report <reporting/tax-report>`, and verify that all tax information is correct.
+- Reconcile all accounts on the :ref:`balance sheet <reporting/balance-sheet>`:
 
-  - Year-end manual adjustments, using the **Adviser Journal Entries**
-    menu (For example, the **Current Year Earnings** and **Retained
-    Earnings** reports).
+  - Update the bank balances in Odoo according to the actual balances found on the bank statements.
+  - Reconcile all transactions in the cash and bank accounts by running the :ref:`aged receivables
+    <reporting/aged-receivable>` and :ref:`aged payables <reporting/aged-payable>` reports.
```

Comment:
maybe we can provide additional info: reconcile how, with which account?

Reply:
Again, any suggestions?